packages/analytics-js-integrations/src/integrations/CommandBar/browser.js

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable class-methods-use-this */
+import {
+  NAME,
+  DISPLAY_NAME,
+} from '@rudderstack/analytics-js-common/constants/integrations/CommandBar/constants';
+import Logger from '../../utils/logger';
+import { loadNativeSdk } from './nativeSdkLoader';
+
+const logger = new Logger(DISPLAY_NAME);
+
+class CommandBar {
+  constructor(config, analytics, destinationInfo) {
+    if (analytics.logLevel) {
+      logger.setLogLevel(analytics.logLevel);
+    }
+    this.analytics = analytics;
+    this.orgId = config.orgId;
+    this.name = NAME;
+    ({
+      shouldApplyDeviceModeTransformation: this.shouldApplyDeviceModeTransformation,
+      propagateEventsUntransformedOnError: this.propagateEventsUntransformedOnError,
+      destinationId: this.destinationId,
+    } = destinationInfo ?? {});
+  }
+
+  init() {
+    loadNativeSdk(this.orgId);
+  }
+
+  isLoaded() {
+    return !!(window.CommandBar && typeof window.CommandBar === 'object');
```

Comment:
```suggestion
    return window.CommandBar && typeof window.CommandBar === 'object';
```

Reply:
Moreoever, the checking condition is not correct. `window.CommandBar` is always an object.
There must be something else we can use to check.

Reply:
I checked commandbar documentation to check for some other function like isReady or isLoaded but nothing

Reply:
Try to check the contents of `window.CommandBar` before and after the load and check if it reveals anything.

Otherwise, let's check with their team.

Reply:
Ok

Reply:
This is done